content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -32,19 +32,24 @@ mitigate some future vulnerabilities too.
 {{% thirdparty-content single="true" %}}
 <!-- if adding another runtime in the future, omit the single setting -->
 
-This is a Linux only feature. In addition, support is needed in the 
+This is a Linux only feature. In addition, support is **needed** in the 
```

Comment:
That change doesn't line up with the PR title, and I also disagree that it's required. Separately, I think it could be OK to change the wording to help people who don't read documentation understand the requirement. But not in this commit.

Reply:
removed